core/controllers/base_test.py

```diff
@@ -279,11 +279,23 @@ def test_alphabetic_i18n_keys(self):
         for filename in filenames:
             with open(os.path.join(os.getcwd(), 'assets', 'i18n', filename),
                       mode='r') as f:
-                next_line = next(f)
-                for line in f:
-                    line = line.replace("{", "")
-                    line = line.replace("}", "")
-                    self.assertGreater(next_line, line)
+                lines = f.readlines()
+                if lines[0] == '{\n':
+                    lines.remove(lines[0])
+                if lines[-1] == '}\n':
+                    lines.pop()
+
+                string_lines = ''.join(lines)
```

Comment:
I don't understand why we're doing all this manipulation. Couldn't you just do something along the lines of:

    key_list = [line[:line.find(':')].strip() for line in lines]
    for key in key_list:
        self.assertTrue(key.startswith('I18N_')

instead of the next 9 lines or so? Instead, you appear to be getting rid of a whole bunch of characters, splitting by spaces, and then finding tokens which start with `I18N_`, which seems quite a lot more convoluted. Generally speaking, tests should be simple, though they can be comprehensive -- you're trying to check that the files are formatted correctly and you can assert that the keys are capitalized, they start with the correct substring, etc.

Also, commented-out code should always be fully removed prior to a commit.